csrc/executor.cpp

```diff
@@ -1141,11 +1461,14 @@ void validateCooperativeLaunch(
   CUDA_SAFE_CALL(cuOccupancyMaxActiveBlocksPerMultiprocessor(
       &num_blocks_per_SM,
       kernel,
-      (int)(launch_params.bdimx() * launch_params.bdimy() * launch_params.bdimz()),
+      (int)(launch_params.bdimx() * launch_params.bdimy() *
+            launch_params.bdimz()),
       (size_t)launch_params.smem()));
 
   TORCH_INTERNAL_ASSERT(
-      (int64_t)(num_blocks_per_SM * at::cuda::getDeviceProperties(device_index)->multiProcessorCount) >=
+      (int64_t)(num_blocks_per_SM *
+                at::cuda::getDeviceProperties(device_index)
+                    ->multiProcessorCount) >=
```

Comment:
I am not changing anything, this is cause by my clang-format. Will fix before merge.